_includes/releases/v23.1/v23.1.0-alpha.9.md

```diff
@@ -0,0 +1,80 @@
+
+<h3 id="v23-1-0-alpha-9-sql-language-changes">SQL language changes</h3>
+
+- The new `prepared_statements_cache_size` [cluster setting](cluster-settings.html) helps to prevent [prepared statement](savepoint.html#savepoints-and-prepared-statements) leaks by automatically deallocating the least-recently-used prepared statements when the cache reaches a given size. [#99254][#99254]
+
+<h3 id="v23-1-0-alpha-9-operational-changes">Operational changes</h3>
+
+- The new `COCKROACH_DISABLE_NODE_AND_TENANT_METRIC_LABELS` [environment variable](cockroach-commands.html#environment-variables) allows you to suppress metrics from a cluster's [Prometheus endpoint](monitoring-and-alerting.html#the-prometheus-endpoint) if they conflict with labels that are applied by external tools that collect metrics from the endpoint. Set the environment variable to a comma-separated list of key-value pairs. [#99820][#99820]
+
+<h3 id="v23-1-0-alpha-9-db-console-changes">DB Console changes</h3>
+
+- The [Index Details](ui-databases-page.html#index-details) now displays the list of most-frequently-used index fingerprints to all users, rather than only to admin users, because the page now queries a view rather than a system table directly. [#99485][#99485]
+- When you search or filter within the [Statements page](ui-statements-page.html) or [Transactions Page](ui-transactions-page.html), if you interactively sort the results using a column that was not part of the original query, a warning displays if you are viewing only a subset of the results, along with a suggestion to update the original query. [#99795][#99795]
+
+<h3 id="v23-1-0-alpha-9-miscellaneous">Miscellaneous</h3>
+
+- Several computed columns have been added to `crdb_internal.statement_statistics_persisted` and `crdb_internal.transaction_statistics_persisted` views and indexed in the system tables referenced by the views:
+
+    - `execution_count`
+    - `service_latency`
+    - `cpu_sql_nanos`
+    - `contention_time`
+    - `total_estimated_execution_time`
+    - `p99_latency`
+
+    [#99417][#99417]
+
+<h3 id="v23-1-0-alpha-9-bug-fixes">Bug fixes</h3>
+
+- Fixed pagination bugs when searching or filtering within the [Databases](ui-databases-page.html) or viewing the details of a database. [#99513][#99513]
+- Fixed a rare bug introduced in v22.2.0 that could cause a node to crash with an `attempting to append refresh spans after the tracked timestamp has moved forward` error when querying the `pg_catalog` and `crdb_internal` virtual tables. If you are experiencing this bug, set the `sql.distsql.use_streamer.enabled` [cluster setting](cluster-settings.html) to `false` before upgrading a cluster to v23.1. [#99443][#99443]
```

Comment:
We have dedicated pages for `pg_catalog` and `crdb_internal` - suggest linking to those pages.